docs/en/setup/backend/logs-template.json

```
@@ -0,0 +1,82 @@
+{
+    "timestamp": {
+        "type": "sequence",
+        "min": "1649643929000",
+        "max": "1649653929000"
+    },
+    "serviceName": {
+        "type": "randomString",
+        "length": "20",
+        "prefix": "test_svc_name_",
+        "letters": true,
+        "numbers": true
+    },
+    "serviceInstanceName": {
+        "type": "randomString",
+        "length": "20",
+        "prefix": "test_svc_inst_name_",
+        "letters": true,
+        "numbers": true
+    },
+    "endpointName": {
+        "type": "randomString",
+        "length": "20",
+        "prefix": "test_endpoint_",
+        "letters": true,
+        "numbers": true
+    },
+    "traceId": {
+        "type": "randomString",
+        "length": "20",
+        "prefix": "test_trace_id_",
+        "letters": true,
+        "numbers": true
+    },
+    "traceSegmentId": {
+        "type": "randomString",
+        "length": "20",
+        "prefix": "test",
+        "letters": true,
+        "numbers": true
+    },
+    "spanId": {
+        "type": "randomInt",
+        "min": "0",
+        "max": "5"
+    },
+    "contentType": {
+        "type": "fixedInt",
+        "value": "1"
+    },
+    "content": {
+        "type": "randomString",
+        "length": "10",
+        "prefix": "test",
+        "letters": true,
+        "numbers": true
+    },
+    "error": {
+        "type": "randomBool"
```

`randomBool` is not good enough for `error` in testing. We need specific success rate

```suggestion
        "type": "randomBool"
        "successRate": "90" # 90% is successful
```